src/main/java/frc/robot/Robot.java

```diff
@@ -60,6 +59,10 @@ public void robotInit() {
 	 */
 	@Override
 	public void robotPeriodic() {
+		RobotContainer.LEDS.indicateNote();
```

Comment:
Please check is enabled 
https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/RobotBase.html#isEnabled()